lib/youtube-dl.js

```diff
@@ -163,25 +163,37 @@ function call(video, args1, args2, options, callback) {
  * @param {Array.<String>} data
  */
 function filterData(data) {
-  var format = data[data.length - 1].split(' - ');
+  var info = JSON.parse(data);
+  var format = info.format.split(' - ');
+  function formatDuration(duration) {
```

Comment:
can you still add this function to util.js? so that it's not created every time `filterData()` is called.

also I just realized a bunch of those functions in util.js are no longer used.